pkg/chunk/cassandra/table_client.go

```diff
@@ -17,7 +17,7 @@ type tableClient struct {
 
 // NewTableClient returns a new TableClient.
 func NewTableClient(ctx context.Context, cfg Config) (chunk.TableClient, error) {
-	session, err := cfg.session()
+	session, err := cfg.session("table_manager")
```

Comment:
Just for consistency. Looks other client names are all dash separated.

```suggestion
	session, err := cfg.session("table-manager")
```